main/tasks/stratum_task.c

```diff
@@ -18,7 +18,7 @@
 
 static const char *TAG = "stratum_task";
 static ip_addr_t ip_Addr;
-static bool bDNSFound = false;
+static int8_t bDNSFound = 0;
```

Comment:
this variable is no longer a bool, give it a clearer name and use and enum for the different states or have a second bool for tracking dns progress